pkg/apis/hive/v1alpha1/clusterdeployment_types.go

```diff
@@ -247,6 +247,9 @@ const (
 
 	// ProvisionFailedCondition indicates that a provision failed
 	ProvisionFailedCondition ClusterDeploymentConditionType = "ProvisionFailed"
+
+	// SyncSetFailureCondition indicates if any syncset for a cluster deployment failed
+	SyncSetFailureCondition ClusterDeploymentConditionType = "false"
```

Comment:
you typically want a human readable text here "SyncSetFailed" (or something like that)

Reply:
Changed to `SyncSetFailedCondition` to keep the name consistent with other condition names.

Reply:
fixed it.